### DISCUSSION TITLE
Disable multiple email addresses

### DISCUSSION THREAD

**Discussion by vuode** [accounts-js/accounts#1179] [Q&A] [answered]
Greetings, accounts-js community!

I was wandering if there is a way to turn off the option of adding more than one email address? I tried using `validateEmail`, but couldn't find a way to check if the user has already provided one email address, only to validate new one. 

Thanks!

**Comment by pradel** [comment, marked as answer]
Hey, this is not possible at the moment.

**Reply by vuode** [reply]
Thank you very much for the answer, hope this option will be available someday.

**Comment by darkbasic** [comment]
Please open a feature request for this so we can track it, I think it would be useful.
